packages/rocketchat-i18n/i18n/en.i18n.json

```
@@ -1310,6 +1310,8 @@
   "error-role-in-use": "Cannot delete role because it's in use",
   "error-role-name-required": "Role name is required",
   "error-room-is-not-closed": "Room is not closed",
+  "Error_Site_URL": "Invalid Site_Url",
+  "Error_Site_URL_description": "Please, update your \"Site_Url\" setting on database as the same url you are accessing your instance then reload the application",
```

```suggestion
  "Error_Site_URL_description": "Please, update your \"Site_Url\" setting find more information <a href=\"https://go.rocket.chat/i/invalid-site-url\">here</>",
```

Not sure actually how links work in i18n... so this might not be right.  The actual url is waiting on PR in docs repo

---

link live 👍